### ISSUE TITLE
.get_mbid returns wrong value for tracks with no MBID

### ISSUE THREAD

**Issue by moustachio** [pylast/pylast#146]
I haven't confirmed this for other methods (e.g. `get_album()`), but when calling `.get_mbid()` on a track that has no listed MBID, the MBID of the corresponding *artist* will be returned (with no warning that you're getting the wrong data). For example, `network.get_track('static-x','set it off').get_mbid()` returns `'8ac9e35c-e92d-4030-99ef-ba4127b1555c'`, but this is the *artist* MBID. We can confirm this by checking the track.getInfo api method directly (at `http://ws.audioscrobbler.com/2.0/?method=track.getInfo&api_key=YOURAPIKEY&artist=static-x&track=set+it+off`):

```
<lfm status="ok">
  <track>
     <name>Set It Off</name>
     <url>http://www.last.fm/music/Static-X/_/Set+It+Off</url>
     <duration>0</duration>
     <streamable fulltrack="0">0</streamable>
     <listeners>39822</listeners>
     <playcount>210863</playcount>
     <artist>
          <name>Static-X</name>
          <mbid>8ac9e35c-e92d-4030-99ef-ba4127b1555c</mbid>
         <url>http://www.last.fm/music/Static-X</url>
     </artist>
...
```

As you can see, there is not track MBID listed, but the corresponding artist MBID is, and that's what `.get_mbid()` is erroneously returning.

The proper behavior, I assume, should be to return `None`.

**Comment by hugovk** [comment]
`None` would make sense in this case. I'm guessing pylast is just looking for the first MBID instead of getting the one from `<track>`.

http://www.last.fm/api/show/track.getInfo

**Comment by hugovk** [comment]
Fixed in https://github.com/pylast/pylast/pull/149.

**Comment by hugovk** [comment]
Released in pylast 1.5.1.
